tutorial/tutorial-gabc-01.html

```diff
@@ -282,11 +282,11 @@
 <img src="kyrie-xvii-goal.png"/>
 </div>
 
-<p>There are still some differences.  Of course, the typefaces are different. The examples above from my system use the TeX typeface "newcent" (New Century Schoolbook), which is very readable. I'll do an experiment with a different typeface later.</p>
+<p>There are still some differences.  Of course, the typefaces are different. The examples above from my system use the <span class="tex">T<span class="epsilon">e</span>X</span> typeface "newcent" (New Century Schoolbook), which is very readable. I'll do an experiment with a different typeface later.</p>
 
 <p>There is also a difference in the hyphenation of the text. The gregorio software did not automatically add a hyphen after the first syllable of <i>KYrie</i>, because there was enough space between the neumes without a hyphen. Feel free to add a hyphen manually in your gabc file if you find that it makes the score more readable for you.</p>
 
-<p>Let's go on to <a href="tutorial-gabc-02.html">another page</a>, to work on the remainder of this Kyrie.</p>
+<p>Let's go on to <a href="./tutorial-gabc-02.html">another page</a>, to work on the remainder of this Kyrie.</p>
```

Comment:
Is there some reason the `./` is necessary here?  The old line was a relative reference already.  If there is a problem without the `./`, then perhaps my comment on the index file is wrong...